WFP-VAM/prism-app
# [DRAFT]: Divided AA into drought and storm layers

### Description


- [Create another type of anticipatory action : Anticipatory Action Storm]
- [ Update code to handle multiple anticipatory actions]
- [Update nav items to handle childrens, add menu selection item on click]
- [Separates AAStorm panel and AADrought panel]


## How to test the feature:
- [Click on the anticipatory action menu to switch between AAStorm and AADrought layers]
- [Panels should be differents]


## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
